### PR TITLE
Include technical contact email field to the updates page

### PR DESCRIPTION
**Pull request by l3ku**
Added a field where the customer can set site-specific technical contact
email addresses, which are then updated via SWD API to the site info.

### CODE REVIEW

**Comment by l3ku** [comment]
Can be freely tested at https://benchmark.fal-test.wp-palvelu.fi/wp-admin/tools.php?page=updates_page.

File: languages/seravo-fi.po [hunk 1]
```diff
@@ -2,14 +2,15 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Seravo Plugin\n"
 "Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/seravo-plugin\n"
-"POT-Creation-Date: 2017-12-07 22:23+0200\n"
-"PO-Revision-Date: 2017-12-07 22:30+0200\n"
-"Last-Translator: Otto Kekäläinen <otto@seravo.com>\n"
+"POT-Creation-Date: 2017-12-12 14:18:49+00:00\n"
+"PO-Revision-Date: 2017-12-12 16:36+0200\n"
+"Last-Translator: Leo Toikka <leo@seravo.fi>\n"
+"Language-Team: Onni Hakala <onni@koodimonni.fi>\n"
```
Comment (ottok) [inline languages/seravo-fi.po]: I wonder what introduces this line..

File: languages/seravo-fi.po [hunk 2]
```diff
@@ -286,11 +287,36 @@ msgstr ""
...
 
-#: lib/updates-page.php:35
+#: lib/updates-page.php:39
 msgid "Seravo updates enabled"
 msgstr "Seravon päivitykset sallittu"
 
-#: lib/updates-page.php:36
+#: lib/updates-page.php:41
+msgid "Technical contact email addresses"
+msgstr "Sivuston tekniset kontaktihenkilöt"
```
Comment (ottok) [inline languages/seravo-fi.po]: yhteyshenkilöt? yhteysosoite? tekninen ilmoitusosoite?

File: lib/updates-page.php [hunk 3]
```diff
@@ -15,24 +15,37 @@
...
+}
+
 ?>
 
   <p><?php _e('Seravo\'s upkeep service includes that your WordPress site is kept up-to-date with quick security updates and regular tested updates of both WordPress core and plugins. If you want full control of updates yourself, you can opt-out from Seravo updates.', 'seravo') ?></p>
 
-  <form name="toggle_seravo_updates" action="<?php echo esc_url( admin_url('admin-post.php') ); ?>" method="post">
-    <?php wp_nonce_field( 'toggle-seravo-updates-on-or-off' ); ?>
+  <form name="seravo_updates_form" action="<?php echo esc_url( admin_url('admin-post.php') ); ?>" method="post">
+    <?php wp_nonce_field( 'seravo-updates-nonce' ); ?>
```
Comment (ottok) [inline lib/updates-page.php]: Great!

File: languages/seravo-fi.po (hunk 1 above)
Comment (l3ku) [reply languages/seravo-fi.po]: This line was somehow in Poedit catalogue properties, no clue as to how it got there.

File: languages/seravo-fi.po (hunk 2 above)
Comment (l3ku) [reply languages/seravo-fi.po]: Changed to "Sivuston tekniset yhteyshenkilöt".

**Comment by ottok** [comment]
Perfect! Well done @l3ku

**Pull request merged**